pkgs/applications/science/electronics/vivado/default.nix

```
@@ -0,0 +1,127 @@
+{ stdenv
+, lib
+, coreutils
+, fetchurl
+, patchelf
+, procps
+, makeWrapper
+, requireFile
+, ncurses5
+, zlib
+, libuuid
+, libSM
+, libICE
+, libX11
+, libXrender
+, libxcb
+, libXext
+, libXtst
+, libXi
+, glib
+, freetype
+, gtk2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vivado";
+  version = "2017.2";
+
+  buildInputs = [
+    patchelf
+    procps
+    ncurses5
+    makeWrapper
+    coreutils
+  ];
```

```suggestion
  nativeBuildInputs = [
    makeWrapper
  ];

  buildInputs = [
    procps
    ncurses5
  ];
```
the other stuff is part of stdenv